src/yaxunit/features/testMessageMarkers.ts

```
@@ -1,7 +1,8 @@
 import { TestsModel } from '../test-model'
-import { editor, MarkerSeverity } from 'monaco-editor-core'
+import { editor, MarkerSeverity, Uri } from 'monaco-editor-core'
 import { TestModelRender } from '../interfaces'
 import { Error } from '../test-model/report'
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)